lib/msf/core/exploit/remote/ldap.rb

```diff
@@ -8,18 +8,23 @@
 
 module Msf
   module Exploit::Remote::LDAP
+    include Msf::Auxiliary::Report
+    include Msf::Exploit::Remote::Kerberos::AuthOptions
```

Comment:
```suggestion
```

We'll be circling back round to unifying the options, so this mixin is gone now 🕵️